### PR TITLE
Fixed UTC time issue while generating a Receipt

### PR DESCRIPTION
**Pull request by faizhashmiRnS**
## Description

This PR will solve: 
1. #917

### CODE REVIEW

File: wallet/lib/pages/purchase_item/purchase_item_screen.dart
Comment (mustafapylons) [inline wallet/lib/pages/purchase_item/purchase_item_screen.dart:747]: @faizhashmiRnS I thought we agreed to use UTC time. This is using the local time now. And with the previous code it wasn't converting to UTC time anyways.

**Review by mustafapylons — COMMENTED** [review]
Need more review before approval.

**Review by faddat — APPROVED** [review]

File: wallet/lib/pages/purchase_item/purchase_item_screen.dart
Comment (kjawadDeveloper2) [reply wallet/lib/pages/purchase_item/purchase_item_screen.dart:747]: @mustafapylons agreed.

**Review by kjawadDeveloper2 — CHANGES_REQUESTED** [review]
UTC functionality will remain.

File: wallet/lib/pages/purchase_item/purchase_item_screen.dart
Comment (faizhashmiRnS) [reply wallet/lib/pages/purchase_item/purchase_item_screen.dart:747]: @mustafapylons  Oh Alright. Let me look into it. Thanks

**Review by kjawadDeveloper2 — APPROVED** [review]

**Comment by StefanB90** [comment]
@faisalnaveedRNS does the recipe have the capability of saving the creation time?

File: wallet/lib/pages/purchase_item/purchase_item_screen.dart
Comment (kjawadDeveloper2) [reply wallet/lib/pages/purchase_item/purchase_item_screen.dart]: @faizhashmiRnS  Sorry but i am just seeing it now. If you see the code the dateTime.now means that we are not getting it or using it from the chain.

**Review by kjawadDeveloper2 — CHANGES_REQUESTED** [review]
Show exact create date.

File: wallet/lib/pages/purchase_item/purchase_item_screen.dart
Comment (faizhashmiRnS) [reply wallet/lib/pages/purchase_item/purchase_item_screen.dart]: > @faizhashmiRnS Sorry but i am just seeing it now. If you see the code the dateTime.now means that we are not getting it or using it from the chain.

Hi, I just had a talk with Sir Faisal. Backend team will work on it.

**Review by faddat — APPROVED** [review]

**Comment by faddat** [comment]
I think fixes are in here.

**Comment by kjawadDeveloper2** [comment]
@faizhashmiRnS  test it on faisal test chain and let me know . Make sure it works on our current chain also. (Doesn't break if it has no date time)

**Comment by faizhashmiRnS** [comment]
> @faizhashmiRnS test it on faisal test chain and let me know . Make sure it works on our current chain also. (Doesn't break if it has no date time)

Alright.

**Review by kjawadDeveloper2 — APPROVED** [review]

**Pull request merged**
